indico/modules/rb/statistics.py

```diff
@@ -30,8 +30,8 @@ def calculate_rooms_booked_time(rooms, start_date=None, end_date=None):
     # Reservations on working days
     reservations = Reservation.find(Reservation.room_id.in_(r.id for r in rooms),
                                     db.extract('dow', ReservationOccurrence.start_dt).between(1, 5),
-                                    ReservationOccurrence.start_dt >= start_date,
-                                    ReservationOccurrence.end_dt <= end_date,
+                                    ReservationOccurrence.start_dt >= datetime.combine(start_date, time.min),
+                                    ReservationOccurrence.end_dt <= datetime.combine(end_date, time.max),
```

Comment:
I'm not sure that this is the best solution. You're accepting a `datetime` but then just ignoring its time part. It seems a bit misleading to me.
Maybe we should use a `date` instead of `datetime` for `start_date` and `end_date`? This way it will be clear that time isn't taken into account.